coreclient/src/clients/update_key.rs

```diff
@@ -101,7 +101,7 @@ mod update_key_flow {
             let owner_domain = conversation.owner_domain();
             let ds_timestamp = api_clients
                 .get(&owner_domain)?
-                .ds_update(params, signer, group.group_state_ear_key())
+                .ds_group_operation(params, signer, group.group_state_ear_key())
```

Comment:
This can probably be removed because it's done by https://github.com/phnx-im/infra/pull/658